templates/advantage/_table.html

```diff
@@ -0,0 +1,86 @@
+
+<div class="row">
+  <div class="table" id="enterprise-table">
+    <div class="row">
+      <div class="col-12 u-sv3" style="overflow-x: auto;">
+        <table class="p-table" style="width: 100%;">
```

Comment:
A `<table>` element is `width: 100%` by default in Vanilla.

Reply:
It's gone